packages/oscal-react-library/src/components/OSCALProperties.tsx

```diff
@@ -30,21 +30,40 @@ const OSCALPropertiesButton = styled(Button)(
 );
 
 /**
- * Accumulates all instances of third-party namespaces into a list.
+ * Sorts the namespaces by NIST namespaces first, orders third-party namespaces alphabetically,
+ * then orders by name.
  *
  * @param {Property[]} properties A list of properties
- * @returns {string[]} A list of third-party namespaces
+ * @returns {Property[] | undefined} A list of third-party namespaces
  */
-function getThirdPartyNamespaces(properties?: Property[]) {
-  const items: string[] = [""];
+function sortProperties(properties?: Property[]): Property[] | undefined {
+  const namespaceSortedProperties = properties?.sort((propA, propB) =>
+    namespaceOf(propA.ns) > namespaceOf(propB.ns) ? 1 : -1
+  );
+  const nistNamespaceFirstProperties = namespaceSortedProperties?.sort((propA, propB) =>
+    isNistNamespace(propA.ns) && !isNistNamespace(propB.ns) ? 1 : -1
+  );
+  const nameSortedProperties = nistNamespaceFirstProperties?.sort((propA, propB) =>
+    namespaceOf(propA.ns) === namespaceOf(propB.ns) && propA?.name > propB?.name ? 1 : -1
+  );
+
+  return nameSortedProperties;
```

Comment:
This is... complicated. And I still _really_ do not like `isNistNamespace`'s existence.

Reply:
I thought that helped with readability of the sorts. Do you have a better suggestion? Also, didn't realize you don't like `isNistNamespace` - thought it would be helpful to create a reusable function if we're making the validation more than once.